config/locales/views/annotation_categories/en.yml

```diff
@@ -25,3 +25,4 @@ en:
     no_annotations: "There are no annotations in this category"
     none: "There are no annotation categories yet."
     one_time_only: "One Time Only"
+    flexible_criterion_choice: 'Associated Criterion (Flexible type only): '
```

Comment:
This is the right string, but don't add it here. Instead, add it as an _attribute_ to the _model_ locale file (look at how this is specified for the annotation category name).